hathor/p2p/states/ready.py

```diff
@@ -57,16 +63,32 @@ def __init__(self, protocol: 'HathorProtocol') -> None:
         # Minimum round-trip time among PING/PONG.
         self.ping_min_rtt: float = inf
 
+        # The last blocks from the best blockchain in the peer
+        self.best_blockchain: list[HeightInfo] = []
```

Comment:
I prefer for this to have 'remote' in the name.

Reply:
We usually use the `peer_` prefix. So, in this case, it would be `peer_best_blockchain`.

Reply:
Pronto:
513af820bfd27578f345d4cee37f823f23ae2a08